00-open-a-shapefile.Rmd

```diff
@@ -78,237 +75,254 @@ Working With Shapefile Attributes in R </a>
 4. <a href="{{ site.baseurl }}/R/crop-extract-raster-data-R/">
 Crop and extract raster values in R</a>
- 
-</div>
 
-NOTE: The data used in this tutorial were collected at Harvard Forest which is
-a the National Ecological Observatory Network field site 
-<a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">
-More about the NEON Harvard Forest field site</a>. These data are a proxy of the 30 year 
-time-series that will be collected from the NEON flux tower and made available
-<a href="http://data.neoninc.org" target=_"blank">from the NEON data portal</a>.
-{: .notice}
- 
-###Notes about R Libraries
-
-We will use the `rgdal` library (package) to work with vector data in R. Notice that `sp`
-is automatically loaded when `rgdal` is loaded. We will use the `raster`
-library to work with rasters. The `rgdal` library also allows us to explore
-metadata using similar commands with both rasters and vectors.
-
-```{r load-libraries }
-
-#load required libraries
-library(rgdal)
-library(raster)
-
-```
+###Additional Resources
 
+</div>
 
 #About Vector Data
-Vector data are composed of discrete geometric locations (x,y values) that define object vertices. The organization of the geometric locations, determines the type of 
-vector that we are working with (point, line or polygon) as follows: 
-
-* **Points:** In a point vector dataset, each point is represented by a single x,y
-coordinate location. There can be many points in a vector point file. Examples:
-plot locations or the locations of individual trees.
-* **Lines:** Lines are composed of many points that are connected. For instance,
-a road or a stream is a series of straight line segments, each bend in the road
-or stream represents a vertex that has an x,y location associated with it. A
-line consists of AT LEAST 2 vertices that are connnected.
+Vector data are composed of discrete geometric locations (x,y values) that
+define object vertices. The organization of the geometric locations, determines
+the type of vector that we are working with (point, line or polygon). 
+
+<figure>
+    <a href="http://geographerswasaut.blogspot.com/2015/01/the-geographical-information-system.html">
+    <img src="http://4.bp.blogspot.com/-e2352F7HlwM/VKa8xr1ZPcI/AAAAAAAAAXk/ETV0gQVF9dc/s1600/4.png"></a>
+    <figcaption>Vector data can be a point, line, or polygon.  Image Source: E. Chrisant.</figcaption>
+</figure>
+
+* **Points:** Each point, a vertex, is represented by a single x,y coordinate
+location. There can be many points in a vector point file. Examples: sampling
+locations or the location of individual trees.
+* **Lines:** Lines are composed of many (at least 2) vertices, or points, that
+are connected. For instance, a road or a stream is a series of straight line
+segments, each bend in the road or stream represents a vertex that has an x,y
+location associated with it.
 * **Polygons:** A polygon consists of 3 or more vertices that are connected and 
```

Comment:
Ok.  I added this figure and rendered the posts to local host to see how it looks.  I'm not a huge fan with this graphic as is,  it shows curved lines with only a few points, instead of the fact that curved lines are really just a long line of short strait lines between vertices.  Thoughts?